projects/commudle-admin/src/app/app.component.ts

```diff
@@ -53,5 +56,22 @@ export class AppComponent {
     this.router.navigate(['/']);
   }
 
+  ngOnInit(){
+     let cc = window as any;
```

Comment:
The same code is added to the cookie consent component?

Reply:
Yes, would remove that. Will only be keeping the component one